stream_alert/rule_processor/rules_engine.py

```diff
@@ -284,7 +284,10 @@ def process_rule(record, rule):
             (bool): The return function of the rule
         """
         try:
-            rule_result = rule.rule_function(record)
+            if rule.context:
+                rule_result = rule.rule_function(record, rule.context)
+            else:
+                rule_result = rule.rule_function(record)
         except Exception:  # pylint: disable=broad-except
```

Comment:
@javuto I know this isn't part of your changes, but can you get rid of this `except Exception:` line and just have this be `try: ... except: ... `. This would get rid of that pylint disable message too